srv/GetGroupUrdf.srv

```diff
@@ -0,0 +1,4 @@
+string group_name # Name of the planning group for which an URDF is requested
+---
+bool success   # indicate successful run of the service
+string urdf_string # URDF of the requested group. Empty if request was not successful e.g. because the group does not exist
```

Comment:
Not sure if there's a preferred style here, but would this read better if the comments were before the fields, and with spaces? e..g

```
# Name of the planning group for which an URDF is requested.
string group_name
---
# Indicate successful run of the service.
bool success

# URDF of the requested group. Empty if request was not successful e.g. because the group does not exist.
string urdf_string 
```